updatecli/install/action.yml

```diff
@@ -6,4 +6,4 @@ runs:
   using: composite
   steps:
     - name: Install Updatecli in the runner
-      uses: updatecli/updatecli-action@9a37c7e35598d7b37d8e7568b40ed9538112be01 # v0.76.1
+      uses: updatecli/updatecli-action@4922d58d06f0d096428933f153275e15a3eb49de # v0.76.1
```

Comment:
I think I messed up with the comment with the version, since it's not pointing to the correct one but the updatecli binary version rather than the updatecli-action version

Reply:
```suggestion
      uses: updatecli/updatecli-action@4922d58d06f0d096428933f153275e15a3eb49de # v2.59.0
```